src/components/Error/index.js

```diff
@@ -1,10 +1,10 @@
-import Grid from '@material-ui/core/Grid';
-import Typography from '@material-ui/core/Typography';
 import PropTypes from 'prop-types';
```

Comment:
What's the difference between `import PropTypes from 'prop-types';` and `import { PropTypes } from 'prop-types';`? I see a similar thing on the next line, where React is not in braces.

Reply:
Good question! It's a counterpart to the two ways  things can be exported. The ES6 import/export syntax allows you to export code from a file in two ways: `export variable` and `export default variable`.

The former can be repeated multiple times, so you can export many things from one file. The latter, default export can only be done once in a file. The default export allows you to set some default value for the file that is exported, unsurprisingly, _by default_. 

When you call `import Component from 'moduleName/Path/To/Component'`, you're importing whatever is exported by default from that file. The other option, `import { Component } from 'moduleName'`, you're importing a specific (non-default) export from that file. The biggest difference is that the default export can be named anything and can be imported by any name, e.g. `import ARBITRARY_NAME_THAT_COULD_BE_ANYTHING from 'moduleName/Path/To/Component'`. The non-default imports require that the name of the imported variable match the name of the exported variable. So if you have `export const SpecificComponentName = ...`, then you have to have `import { SpecificComponentName } from 'moduleName'`. 

In this specific case, the developers of the material-ui library have supported both ways (default and named) for importing their components. To use the default import, you import the exact component you want from a file named for that component (e.g. `import Grid from '@material-ui/core/Grid'`). They only later started supporting named imports where you can import whichever set of components you need from the top-level folder (e.g. `import { Grid, Typography } from '@material-ui/core'`). This used to have some performance implications because of what was essentially a bug that meant the entire library was being imported, but that is no longer the case.